src/lib/components/markdown/example.md

```diff
@@ -0,0 +1,78 @@
+# Headers
```

Comment:
this is a good approach but I'm curious about show code functionality of storybook. do we want to see this usages there? if so, now how it looks {example} or example.md content

Reply:
Yeah, using this example.md file, its contents don't show on the `Show code` feature. However, I added raw markdown and rendered markdown to the example, so you can see how everything is used there.

Reply:
great! thank you for landing this component ✨